instance/tests/integration/integration_instance.py

```diff
@@ -252,7 +253,7 @@ def assert_theme_provisioned(self, instance, appserver, application):
 
         self._assert_theme_favicon_in_html(instance, application, favicon_url)
 
-    @shard(1)
```

Comment:
@viadanna Similar question here, do we need to redistribute integration tests across shards or could we keep the existing distribution?

Reply:
Again, not strictly necessary, but the build time dropped from 2.5 hours to 1.5.

Reply:
@viadanna Definitely fine to keep these changes, then 😄